src/features/tokens/TokenListModal.tsx

```diff
@@ -79,28 +79,44 @@ export function TokenList({
   onSelect: (token: TokenMetadata) => void;
 }) {
   const tokens = useMemo(() => {
-    return getAllTokens().filter((t) => {
-      const q = searchQuery?.trim().toLowerCase();
-      const hasRoute = hasTokenRoute(originCaip2Id, destinationCaip2Id, t.address, tokenRoutes);
-      if (!q) return hasRoute;
-      else
-        return (
-          hasRoute &&
-          (t.name.toLowerCase().includes(q) ||
+    const q = searchQuery?.trim().toLowerCase();
+    return getAllTokens()
+      .map((t) => {
+        const hasRoute = hasTokenRoute(originCaip2Id, destinationCaip2Id, t.address, tokenRoutes);
+        return { ...t, disabled: hasRoute };
+      })
+      .sort((a, b) => {
+        if (!a.disabled && b.disabled) {
+          return 1;
+        } else if (a.disabled && !b.disabled) {
+          return -1;
+        } else {
+          return 0;
+        }
+      })
+      .filter((t) => {
+        if (!q) return t;
+        else {
+          return (
+            t.name.toLowerCase().includes(q) ||
             t.symbol.toLowerCase().includes(q) ||
-            t.address.toLowerCase().includes(q))
-        );
-    });
+            t.address.toLowerCase().includes(q)
+          );
+        }
+      });
   }, [searchQuery, originCaip2Id, destinationCaip2Id, tokenRoutes]);
 
   return (
     <div className="flex flex-col items-stretch divide-y divide-gray-100">
       {tokens.length ? (
         tokens.map((t) => (
           <button
-            className="-mx-2 py-2 px-2 rounded hover:bg-gray-100 active:bg-gray-200 transition-all duration-250"
+            className={`-mx-2 py-2 px-2 rounded  ${
+              t.disabled ? 'hover:bg-gray-200' : 'bg-gray-100 '
+            } transition-all duration-250`}
             key={`${t.caip2Id}-${t.address}`}
             type="button"
+            disabled={!t.disabled}
```

Comment:
This looks reversed? should the new disabled value be the ! of what is is now?

Reply:
Ah! I got it... It was working but the naming is confusing. Reversed this in new commit